src/widgets/mentors/ui/mentors.module.scss

```diff
@@ -0,0 +1,33 @@
+.mentors-container {
+  background: $color-gray-100;
+}
+
+.mentors-content {
+  flex-direction: row;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 40px;
+  }
+}
+
+.mentors-info {
+  display: flex;
+  flex-direction: column;
+  max-width: 640px;
+
+  @include media-tablet-large {
+    width: 100%;
+  }
+}
+
+.picture {
+  max-width: 420px;
+  max-height: 420px;
```

Comment:
I suggest to move it in `.image` & add break points

Reply:
It seems that this can be removed completely. I'll add the break point above for image size.

Reply:
Done. The break point above is added.